trace/tests/unit/ext/flask/app/main.py

```diff
@@ -13,16 +13,36 @@
 # limitations under the License.
 
 import flask
+import mysql.connector
+
 from opencensus.trace.ext.flask.flask_middleware import FlaskMiddleware
+from opencensus.trace import config_integration
 
 app = flask.Flask(__name__)
 middleware = FlaskMiddleware(app)
+config_integration.trace_integrations()
 
 
 @app.route('/')
 def hello():
     return 'hello'
 
 
+@app.route('/mysql')
+def query():
+    conn = mysql.connector.connect(user='root', password='19931228')
```

Comment:
Does this create an in-memory database, or connect to an external database, or do nothing?  Can you add a comment?

If we're adding a dependency that tests require a running mysql database, we'll need to do some work around that.

Reply:
I haven't setup circle ci to run the end-to-end test yet, I just run this application locally to test. So it's fine right now. The database is pre-installed and configured in my local machine.